http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/app/CromwellRunnerAppInstall.scala

```diff
@@ -119,16 +122,20 @@ class CromwellRunnerAppInstall[F[_]](config: CromwellRunnerAppConfig,
         // convention is that the database user is the same as the service account name
         raw"postgres.user=${params.ksaName.value}",
         raw"postgres.dbnames.cromwell=${dbNames.cromwell}",
-        raw"postgres.dbnames.tes=${dbNames.tes}"
+        raw"postgres.dbnames.tes=${dbNames.tes}",
+        raw"postgres.dbnames.cromwellMetadata=${dbNames.cromwellMetadata}"
       )
     } yield Values(values.mkString(","))
 
   override def checkStatus(baseUri: Uri, authHeader: Authorization)(implicit ev: Ask[F, AppContext]): F[Boolean] =
     cromwellDao.getStatus(baseUri, authHeader).handleError(_ => false)
 
   private def toCromwellRunnerAppDatabaseNames(dbNames: List[String]): Option[CromwellRunnerAppDatabaseNames] =
-    (dbNames.find(_.startsWith("cromwell")), dbNames.find(_.startsWith("tes")))
+    (dbNames.find(_.startsWith("cromwell")),
+     dbNames.find(_.startsWith("tes")),
+     dbNames.find(_.startsWith("cromwellmetadata"))
```

Comment:
I would expect `dbNames.find(_.startsWith("cromwell"))` to also include `dbNames.find(_.startsWith("cromwellmetadata"))`. Is there any chance of this causing problems?

Reply:
Since we're in the CROMWELL_RUNNER here, does the other Cromwell DB have a longer name that we can search for more specifically? Or does it not matter?

Reply:
Good questions - Cromwell DB doesn't seem to have a longer name, but maybe we could give it one. 

I don't have a good intuition about whether this could cause problems. I'll investigate...

Reply:
Oh gosh, that's a good catch. This could end up passing the wrong DB to the chart. 🤦 

Thinking..

Reply:
Trying something -- @mspector do you mind if I push a commit to this PR ?

Reply:
not at all, thanks @rtitle

Reply:
Chatted offline with @mspector -- I was thinking of ways to be more generic about these DB names that don't require testing prefixes with `startsWith`. But it's tricky -- will continue to think about this outside the scope of this PR, but @mspector will push the simpler/more direct fix for now.